packages/components/src/components/FunctionChart.tsx

```diff
@@ -56,22 +58,21 @@ export const FunctionChart: React.FC<{
     })
     .filter((x) => x !== null)
     .map(({ x, value }) => {
-      let percentiles = getPercentiles(percentileArray, value);
       return {
         x: x,
-        p1: percentiles[0],
-        p5: percentiles[1],
-        p10: percentiles[2],
-        p20: percentiles[3],
-        p30: percentiles[4],
-        p40: percentiles[5],
-        p50: percentiles[6],
-        p60: percentiles[7],
-        p70: percentiles[8],
-        p80: percentiles[9],
-        p90: percentiles[10],
-        p95: percentiles[11],
-        p99: percentiles[12],
+        p1: unwrap(value.inv(0.01)),
+        p5: unwrap(value.inv(0.05)),
+        p10: unwrap(value.inv(0.12)),
+        p20: unwrap(value.inv(0.20)),
+        p30: unwrap(value.inv(0.30)),
+        p40: unwrap(value.inv(0.40)),
+        p50: unwrap(value.inv(0.50)),
+        p60: unwrap(value.inv(0.60)),
+        p70: unwrap(value.inv(0.70)),
+        p80: unwrap(value.inv(0.80)),
+        p90: unwrap(value.inv(0.90)),
+        p95: unwrap(value.inv(0.95)),
+        p99: unwrap(value.inv(0.99)),
```

Comment:
happy to see .inv() getting used directly

Reply:
Yes! I'm happy for this to